pydictable/core.py

```diff
@@ -106,7 +106,17 @@ def get_fields(cls) -> Dict[str, Field]:
         for name, th in get_type_hints(cls).items():
             if name not in fields:
                 fields[name] = cls.__get_field_by_type_hint(th)
-        return fields
+
+        ordered_fields = {}
+        for name, value in dict(vars(cls)).items():
```

Comment:
Can we only take name from this and add it to ordered fields?

Reply:
The vars() includes any other attributes other than FieldType and even any methods on the class.
Moved the instance check for Field is below, as it was not readable as a single line.

Reply:
Updated